### DISCUSSION TITLE
zh/docs/provider/storage/localFileSystem/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2996] [Docs comments]
# zh/docs/provider/storage/localFileSystem/

使用本地文件系统作为Casdoor的存储提供者

https://casdoor.org/zh/docs/provider/storage/localFileSystem/

**Comment by wzzc-dev** [comment]
domain 在哪里修改

**Comment by yhgroup** [comment]
docker中映射到哪个目录呢？

**Reply by kami-poi** [reply]
You need mount /files to your local directory or volume. Otherwise, you need to give user ```casdoor``` read and write permission on /files directory.
